### PR TITLE
[CON-718] bitwise operators

### PR DESCRIPTION
**Pull request by morus12**

### CODE REVIEW

File: sliceutil/anonymize_test.go [hunk 1]
```diff
@@ -68,3 +68,68 @@ func Test_Anonymize(t *testing.T) {
...
+
+	type testCase[T any] struct {
+		name   string
+		input  []T
+		expect []string
+	}
+
+	type foo string
+
+	t.Run("ints", func(t *testing.T) {
```
Comment (nmarie22) [inline sliceutil/anonymize_test.go:82]: This isn't actually testing ints

**Review by nmarie22 — CHANGES_REQUESTED** [review]

**Review by nmarie22 — DISMISSED** [review]

File: database/qb/expression.go [hunk 2]
```diff
@@ -17,6 +19,7 @@ type expressionUnion struct {
 	value interface{}
 	field *TableField
 	multi []expressionUnion
+	cond  *ConditionExpression
```
Comment (apynes) [inline database/qb/expression.go:22]: this should just be `binaryExpression`

**Review by apynes — CHANGES_REQUESTED** [review]
Some small changes and we need tests for the other query types, specifically Update since that is what the ticket is ultimately for.

File: sliceutil/anonymize.go [hunk 3]
```diff
@@ -9,3 +9,13 @@ func Anonymize[T any](src []T) []interface{} {
 
 	return dst
 }
+
+func String[T ~string](src []T) []string {
```
Comment (apynes) [inline sliceutil/anonymize.go:13]: missing documentation on a public function

Comment (apynes) [reply sliceutil/anonymize.go:13]: This should be called "Strings" or "ToStrings" or "ToStringSlice" or something similar. `.String` means give me a string representation of the object, e.g. `String([]int{1,2,3}) => "[ 1, 2, 3 ]" `

**Review by apynes — CHANGES_REQUESTED** [review]
Minor things

**Review by apynes — APPROVED** [review]

**Review by nmarie22 — APPROVED** [review]

**Pull request merged**